astropy/io/votable/exceptions.py

```diff
@@ -1448,6 +1461,18 @@ class E23(VOTableSpecWarning):
     default_args = ('x',)
 
 
+class E24(VOWarning, ValueError):
+    """
+    Non-ASCII unicode values should not be written when the FIELD datatype="char",
```

Comment:
Nitpick: API doc formatting.
```suggestion
    Non-ASCII unicode values should not be written when the FIELD ``datatype="char"``,
```